janusgraph-core/src/main/java/org/janusgraph/graphdb/configuration/JanusGraphConstants.java

```diff
@@ -73,6 +78,11 @@
             if (!v.isEmpty()) b.add(v);
         }
         COMPATIBLE_VERSIONS = b.build();
+        
+        for (String v : props.getProperty("titan.compatible-versions", "").split(",")) {
```

Comment:
agreed! I'll make the proper changes (there's a conflict as well), submit and squash commits.